docs/photoniq/vwrs/02-get-started-vwrs/get-started-vwrs-web-and-mobile.md

```diff
@@ -0,0 +1,85 @@
+---
+sidebar_position: 10
+title: Integrating VWRs on Web and Mobile
+---
+
+Different types of applications require distinct approaches for integrating with Virtual Waiting Rooms (VWRs) as the applications' platform and user interaction models determine the integration process. 
+
+Let's demonstrate how web and mobile applications can effectively integrate and use VWRs to manage traffic and enhance user experiences.
+
+
+## Prerequisites
+
+Before proceeding, ensure to have the following tasks completed:
+
+- **Get Credentials and Access**: Creating a waiting room requires authentication with an API key. Contact and work with Macrometa personnel to obtain the necessary credentials to proceed.
+- **Determine waiting room needs**: The PhotonIQ VWRs offers numerous features to customize your waiting room needs:
+    - **Queue types**: Depending on your particular business use case, VWR offers different [queue types](https://www.macrometa.com/docs/photoniq/vwrs/queue-types). For example, the FIFO type queues and dequeues users based on their order of arrival, while the random queue admits users randomly. 
+    - **Required URLs**: Determine the URLs to incorporate the waiting rooms.
+- **Gather information**: Gather all the required information, like waiting room details and traffic expectations, to simplify the setup process.
+
+## Integrating on Web 
+
+Web apps, also called non-native or browser-based apps, are typically web-based and use standard web technologies such as HTML, CSS, and JavaScript. These applications run within web browsers and are the most common types integrated with VWRs. Integrating your web applications with VWRs occurs through web-based configurations, making it easier to implement an effective traffic management solution like VWR.
+
+This guide explains the steps to connect your web app to VWRs, enhancing your ability to manage high traffic efficiently and maintain a seamless user experience.
+
+### Steps
+
+To integrate your virtual waiting room with your web application:
```

Comment:
Do we capitalize Virtual Waiting Room?

Reply:
There are times we have these capitalized, and other times they are not.  I think we should be consistent throughout our documentation.